### PR TITLE
Translation AZ 4.4

### PR DESCRIPTION
**Pull request by aytajde**
#17

### CODE REVIEW

File: 4-Classification/4-Applied/translations/README.az.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+- Veb modeli necə yaratmalı və onu Onnx model kimi necə yadda saxlamalı
+- Modeli nəzərdən keçirməklə Netron modeli necə istifadə etməli
+- Veb tətbiqdə nəticə çıxarmaq üçün modeli necə istifadə etməli
+
+## Modelinizi yaradın
+
+Tətbiq olunan MÖ sistemlərinin yaradılması biznes sistemləri üçün bu texnologiyalardan istifadə etməyin vacib hissəsidir. Onnx-dən istifadə etməklə veb proqramlar daxilində modellərdən istifadə etmək olar (ehtiyac olduqda onları oflayn kontekstdə istifadə etmək lazımdır).
+
+[Əvvəlki dərsdə](../../../3-Web-App/1-Web-App/README.md), UFO ilə bağlı regresiya modelini yaradıb, "pickle" edib, və Flask tətbiq daxilində istifadə etmişdik. Bu arxitekturanı bilmək çox faydalı olsa da, tamamilə Phyton proqramıdır. Lakin, sizdən Javascript tətbiqi də yazmaq tələb oluna bilər.
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:23]:
```suggestion
[Əvvəlki dərsdə](../../../3-Web-App/1-Web-App/README.md), UFO ilə bağlı reqresiya modelini yaradıb, "pickle" edib, və Flask tətbiqi daxilində istifadə etmişdik. Bu arxitekturanı bilmək çox faydalı olsa da, tamamilə Python proqramıdır. Lakin, sizdən Javascript tətbiqi də yazmaq tələb oluna bilər.
```

File: 4-Classification/4-Applied/translations/README.az.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+Bu dərsdə nəticə almaq üçün Javascript əsasında veb tətbiq yarada bilərik. Əvvəlcə, modeli öyrətmək və Onnx ilə işləməsi üçün hazırlamaq lazımdır.
+
+## Tapşırıq - klasifikasiya modelini öyrədin
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:27]:
```suggestion
## Tapşırıq - klassifikasiya modelini öyrədin
```

File: 4-Classification/4-Applied/translations/README.az.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+İlk öncə, klasifikasiya modelini təmizlənmiş mətbəxlər data dəstini istifadə etməklə öyrədək
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:29]:
```suggestion
İlk öncə, klassifikasiya modelini təmizlənmiş mətbəxlər data dəstini istifadə etməklə öyrədək
```

File: 4-Classification/4-Applied/translations/README.az.md [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+
+1. onx yaradıb faylı **model.onnx** kimi yadda saxlayaq:
+
+    ```python
+    onx = convert_sklearn(model, initial_types=initial_type, options=options)
+    with open("./model.onnx", "wb") as f:
+        f.write(onx.SerializeToString())
+    ```
+
+    > Konvertasiya skriptində [options](https://onnx.ai/sklearn-onnx/parameterized.html)-nı ötürə bilərik. Bu halda, 'nocl'-nın True və 'zipmap'-ın False olmasını təmin etdik.
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:138]:
```suggestion
    > Konvertasiya skriptində [options](https://onnx.ai/sklearn-onnx/parameterized.html)-ı ötürə bilərik. Bu halda, 'nocl'-nın True və 'zipmap'-ın False olmasını təmin etdik.
```

File: 4-Classification/4-Applied/translations/README.az.md [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+    Bu təsnifat modeli olduğundan, lüğətlərin siyahısını yaradan ZipMap-ı silmək seçimimiz var (vacib deyil). `nocl` modelə daxil edilmiş sinif məlumatlarına istinad edir. `nocl`-i 'True' olaraq təyin edərək modelinizin ölçüsünü azaldaq.
+
+Bütün notebooku işə salmaqla Onnx modelini yaradaraq onu bu qovluqda yadda saxlaya biləcəyik.
+
+## Modelə nəzər yetirək
+
+Onnx modelləri Visual Studio kodunda visual olaraq görünmür. Bir çox tədqiqatçı modelin düzgün qurulmasını təmin etmək üçün onu vizuallaşdırmağa istifadə etdiyi pulsuz proqramlardan istifadə edir. [Netron](https://github.com/lutzroeder/Netron) yükləyək və model.onnx faylını açaq. Sadə modelin 380 girişi və siyahıda göstərilən təsnifatı ilə vizuallaşdırıldığını görə bilərsiniz:
+
+![Netron visualı](images/netron.png)
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:147]:
```suggestion
![Netron vizualı](images/netron.png)
```

File: 4-Classification/4-Applied/translations/README.az.md [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+
+Netron modelləri visual olaraq görmək üçün bir vasitədir.
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:149]:
```suggestion
Netron modelləri vizual olaraq görmək üçün bir vasitədir.
```

File: 4-Classification/4-Applied/translations/README.az.md [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+
+İndi bu səliqəli modeli veb proqramında istifadə etməyə hazırıq. Gəlin, soyuducuya baxan zaman işimizə yarayacaq bir proqram yaradaq və modelə görə müəyyən edilmiş mətbəx növünü bişirmək üçün qalıq inqridiyentlərin hansı kombinasiyasından istifadə edə biləcəyinizi anlamağa çalışaq.
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:151]:
```suggestion
İndi bu səliqəli modeli veb proqramında istifadə etməyə hazırıq. Gəlin, soyuducuya baxan zaman işimizə yarayacaq bir proqram yaradaq və modelə görə müəyyən edilmiş mətbəx növünü bişirmək üçün qalıq inqrediyentlərin hansı kombinasiyasından istifadə edə biləcəyinizi anlamağa çalışaq.
```

File: 4-Classification/4-Applied/translations/README.az.md [hunk 8]
```diff
@@ -0,0 +1,315 @@
...
+            }
+        }
+               
+    </script>
+    ```
+
+Bu kodda eyni anda bir neçə prosses baş verir:
+
+1. Mümkün 380 dəyərdən (1 və ya 0) ibarət massiv yaratdıq və inqrediyentin işarələnib göstərilməsindən asılı olaraq nəticə çıxarmaq üçün modelə göndəriləcək.
+2. Qeyd qutuları massivi və onların proqram başlayanda çağırılan `init` funksiyasında yoxlanılıb-yoxlanılmadığını müəyyən etmək üçün bir üsul yaratdıq. Yoxlama qutusu işarələndikdə, `ingredients` massivi seçilmiş inkrediyenti əks etdirmək üçün yenilənir.
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:283]:
```suggestion
2. Qeyd qutuları massivi və onların proqram başlayanda çağırılan `init` funksiyasında yoxlanılıb-yoxlanılmadığını müəyyən etmək üçün bir üsul yaratdıq. Yoxlama qutusu işarələndikdə, `ingredients` massivi seçilmiş inqrediyenti əks etdirmək üçün yenilənir.
```

File: 4-Classification/4-Applied/translations/README.az.md [hunk 9]
```diff
@@ -0,0 +1,315 @@
...
+## Proqraqmı test edək
+
+index.html faylının yerləşdiyi qovluqda Visual Studio-da yeni terminal açaq və qlobalda quraşdırılmış [http-server](https://www.npmjs.com/package/http-server) mövcud olduğundan əmin olaq. Sorğuda `http-server` yazdıqdan sonra Localhost açılacaq. Beləliklə Veb tətbiqə nəzər yetirə bilərik. Müxtəlif inqridiyentlər əsasında hansı mətbəxin tövsiyə olunduğunu yoxlayaq:
+
+![ingrediyent veb tətbiq](../images/web-app.png)
+
+Təbriklər, 'tövsiyyə edən' veb tətbiq yaratdıq. İndi isə sistemi qurmaq üçün daha çox vaxt ayıraq!
+## 🚀Məşğələ
+
+Hazırki veb tətbiq minimal səviyyədədir. Bu səbəbdən inqridiyentlər və onların indekslərinə əsasən [ingredient_indexes](../../data/ingredient_indexes.csv) datasından istifadə etməklə davam etmək lazımdır. Verilmiş milli yeməyi hazırlamaq üçün hansı dadlardan istifadə edilməlidir, müəyyən edək.
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/README.az.md:301]:
```suggestion
Hazırki veb tətbiq minimal səviyyədədir. Bu səbəbdən inqrediyentlər və onların indekslərinə əsasən [ingredient_indexes](../../data/ingredient_indexes.csv) datasından istifadə etməklə davam etmək lazımdır. Verilmiş milli yeməyi hazırlamaq üçün hansı dadlardan istifadə edilməlidir, müəyyən edək.
```

File: 4-Classification/4-Applied/translations/assignment.az.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
+# Tövsiyyə sistemi yaradın
+
+## Təlimatlar
+
+Bu dərslərdəki təcrübələrə əsasən, Onnx Runtime və ya çevrilmiş bir Onnx modeli istifadə edərək JavaScript əsaslı veb tətbiqi necə yaratmağı artıq bilirik. Bu dərsdəki məlumatlardan və ya başqa mənbədən (zəhmət olmasa, mənbəni göstərməklə) məlumatları istifadə edərək yeni bir tövsiyyəçi yaratmaqla təcrübələr aparın. Fərqli şəxsiyyət xüsusiyyətlərinə görə müəyyən bir ev heyvanı tövsiyyəçisi və ya şəxsin əhvalına uyğun musiqi janrı tövsiyyə edən bir sistem yarada bilərsiniz. Kreativ olun!
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/assignment.az.md:5]:
```suggestion
Bu dərslərdəki təcrübələrə əsasən, Onnx Runtime və ya çevrilmiş bir Onnx modeli istifadə edərək JavaScript əsaslı veb tətbiqi necə yaratmağı artıq bilirik. Bu dərsdəki məlumatlardan və ya başqa mənbədən (zəhmət olmasa, mənbəni göstərməklə) məlumatları istifadə edərək yeni bir tövsiyyəçi yaratmaqla təcrübələr aparın. Fərqli şəxsiyyət xüsusiyyətlərinə görə müəyyən bir ev heyvanı tövsiyəçisi və ya şəxsin əhvalına uyğun musiqi janrı tövsiyə edən bir sistem yarada bilərsiniz. Kreativ olun!
```

File: 4-Classification/4-Applied/translations/assignment.az.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+
+## Rubrika
+
+| Kriteriya | Nümunəvi                                                              | Adekvat                              | İnkişafa ehtiyyacı olan                 |
+| -------- | ---------------------------------------------------------------------- | ------------------------------------- | --------------------------------- |
+|          | Veb tətbiq və ya notbuk verilib Hər ikisi istifadəyə hazırdır. | İkisindən biri əksikdir və ya qüsurludur. | Hər ikisi ya əksikdir, ya da qüsurludur. |
```
Comment (muradtries) [inline 4-Classification/4-Applied/translations/assignment.az.md:11]:
```suggestion
|          | Veb tətbiq və ya notbuk verilib. Hər ikisi istifadəyə hazırdır. | İkisindən biri əskikdir və ya qüsurludur. | Hər ikisi ya əskikdir, ya da qüsurludur. |
```

**Review by muradtries — COMMENTED** [review]
Thanks @aytajde for contribution

File: 4-Classification/4-Applied/translations/assignment.az.md (hunk 11 above)
Comment (aytajde) [reply 4-Classification/4-Applied/translations/assignment.az.md]: @muradtries thanks for detailed review and corrections. all done 👍

**Pull request merged**